### DISCUSSION TITLE
Does the supervision support drawing for pose estimation task ?

### DISCUSSION THREAD

**Discussion by leviethung2103** [roboflow/supervision#199] [Ideas]
### Search before asking

- [X] I have searched the Supervision [issues](https://github.com/roboflow/supervision/issues) and found no similar feature requests.

### Question

Hello,
Does the supervision support drawing for pose estimation task ? 

As far as I know, the supervision currently supports segmentation and detection only ? Correct me if I was wong

**Comment by hardikdava** [comment]
Thanks @leviethung2103 for taking interest in `supervision`. Currently we support for classifiction, object detection and segmentation but we will extend this in future. We will surely think of supporting pose estimation in future. 

Thank you again for sharing your idea.

@SkalskiP , new feature request!

**Comment by SkalskiP** [comment]
Hi @leviethung2103 👋🏻 Thanks a lot for your interest in `supervision`. I like the idea a lot. I want us to support pose estimation at some point. Unfortunately, I can't put any timeline on this task for now. 

Let's convert this question into a discussion and move it to the Ideas section for further refinement.

**Comment by SkalskiP** [comment]
I added this to our backlog https://github.com/orgs/roboflow/projects/5/views/1

**Comment by tobieabel** [comment]
@SkalskiP This would be a great addition to Supervision now that Roboflow allows us to annotate pose estimation projects.  If its of interest, the specific use case I have in mind is for a client who wants to detect when waiting staff are interacting with a table in a restaurant.  I have used object detection and zonal triggering from Supervision to do this for now but using pose estimation instead would be ideal as I'd be able to trigger an interaction only if the staff's hands crossed into the table zone

**Comment by SkalskiP** [comment]
Hi @tobieabel 👋🏻 Adding KeyPoint support is on our to-do list for Q1. https://github.com/roboflow/supervision/issues/452
